<!DOCTYPE html>
<html>
  <head>
    <meta http-equiv="refresh" content="0; URL='http://sailing.cs.cmu.edu/main/genamap'" />
  </head>
  <!--<head>
    <meta charset='utf-8'>
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <link href='https://fonts.googleapis.com/css?family=Architects+Daughter' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/print.css" media="print">

    <title>Genamap by genamap</title>
  </head>-->

  <body>
    <header>
      <div class="inner">
        <h1>Genamap</h1>
        <h2>Scalable and Intuitive Structured Association Mapping</h2>
        <a href="https://github.com/genamap" class="button"><small>Follow me on</small> GitHub</a>
      </div>
    </header>

    <div id="content-wrapper">
      <div class="inner clearfix">
        <section id="main-content">
          <p>GenAMap</p>

<p>An Visual Analytics Software Platform for eQTL and GWAS Analysis</p>

<p>Ross E Curtis and Eric P Xing</p>

<p>Home  Publications  Tutorials/Docs  Three-way analysis Download  </p>

<p>Web Based Service</p>

<p>Description</p>

<p>The proliferation of genomic data has increased the usefulness of complex machine learning algorithms for structured association mapping. Such methods are proved to able to effectively relate genetic polymorphisms with phenotypes, but to use these algorithms correctly, as to avoiding artifacts raised by correlation, typically requires algorithmic expertise, which often constrains users to simpler methods. To overcome it, the GenAMap software platform was developed and released in 2010. Since then, the sizes of available biological data have continued their exponential increase. Another challenge is raised on computational resources. To address this, GenAMap is redesigned for scalability and updated with state-of-the-art methods, allowing efficient calculations on human-scale data. The user experience is an intuitive webapp, where users only need select the task to run for their data and the algorithms on human-scale data will finished in a reasonable time.</p>

<p>Avaliable Algorithms</p>

<p>Single SNP Hypothesis Testing</p>

<p>Chi Squared Tests
Wald Test
Fisher Exact Test
Structured Association Mapping</p>

<p>Linear Regression Standard Linear Regression is available as a simple and fast association mapping algorithm. Lasso and Ridge Regression are also available through relevant parameters.
GcLasso GcLasso is designed to incorporate the correlation structure of multiple quantitative traits such as clinical metrics and gene expressions in association analysis
GwLasso GwLasso is an enhanced version of GcLasso. It not only considers the graph topology of traits, but also consider the weight of edges.
TreeLasso Tree lasso is improved upon GcLasso and GwLasso, that it allows the graph relation to form into tree structure, offers more flexibity in structure information encoded by traits.
Multi-Pop Lasso Multi-Pop Lasso is designed for detecting causal genetic markers through a joint association analysis of multiple populations
Adaptive Multi-Task Lasso Adaptive Multi-Task Lasso takes into account multiple traits simultaneously while weights of different covariates are learned adaptively from prior knowledge and data.
Confounding Factors Correction and Association Mapping</p>

<p>Linear Mixed Model Linear Mixed Model corrects confounding factors before applying Linear Regression for structured association mapping.
Team Members</p>

<p>PI Eric P Xing
Software Architect, scientist, designer, and development director Ross E Curtis</p>

<p>Current Members Haohan Wang, Ben Lengerich, Min Kyung Lee</p>

<p>Maintainance Please send related questions (bug reports) to haohanw at cs dot cmu dot edu</p>

<p>Algorithm Contribution Seyoung Kim, Kriti Puniyani, Seunghak Lee, Junming Yin, Ross E Curtis, Jun Zhu</p>

<p>Software Development Dylan Steele, Bryan Yan, Aditya Gautam, Liuyu Jin, Beilin Li, Flavia Grosan, Anuj Goyal, Jorge Vendries, Michael Zuromskis, Sharath Babu, James Moffatt, Kelly Chan</p>

<p>Special Thanks for the following Open Source Libraries JUNG, SSHTools, JHeatChart, JFreeChart, Mysql++, BLAS, ATLAS, CRAN-R project and libraries, Parallel Spectral Clustering in Distributed Systems, BiNGO, javastat, commons math</p>

<p>Desktop Version</p>

<p>What is GenAMap? GenAMap brings the power of structured association mapping to a usable, intuitive interface. GenAMap makes GWAS and eQTL studies easier on three fronts:</p>

<p>1) Data management - create subsets, manage, and visualize genomic and phenotype data.</p>

<p>2) Algorithms - run algorithms to generate structure such as a gene network or a population stratification. Or, run a structured association algorithm. All algorithms are run on a remote cluster complete with complex parallelization schemes to provide max run-time efficiency.</p>

<p>3) Visualizations - tools to visualize the structure of the data while exploring the association results. We provide visualization tools to get a feel of the overall associations in the dataset, along with the ability to zoom in and explore specific parts of the data. Tools are interactive, linking to databases online for more information.</p>

<p>A) Network visualization, annotation, and exploration
B) The exploration of associations from the genome to complex phenotypic trait networks
C) Population association analysis
D) Three-way association analysis</p>

<p>History
GenAMap 1.0 - released July 2010. Includes data implementation, algorithm control implementation, and first network visualizations for networks and associations. Supports the GFlasso algorithm.
GenAMap 2.0 - released February 2011. Visualization of networks, trees, and population structure. GFlasso fully supported.</p>

<p>GenAMap 3.0 - released September 2011. This is the final release of GenAMap with population, three-way, and network association analysis supported. See docs for details.
(C) SAILING Lab, 2008 Copyright of published papers held by publishing bodies in question 
Last updated 05/30/2016</p>
        </section>

        <aside id="sidebar">


          <p>This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the Architect theme by <a href="https://twitter.com/jasonlong">Jason Long</a>.</p>
        </aside>
      </div>
    </div>

  
  </body>
</html>
